api/api.go

```diff
@@ -210,10 +210,15 @@ func New(
 	verboseLogging uint, authType uint8,
 	opts *ClientOptions) (Client, error) {
 
-	if hostname == "" || username == "" || password == "" || authType > 1 {
+	if hostname == "" || username == "" || password == "" {
 		return nil, errNewClient
 	}
 
+	if authType != 0 && authType != 1 {
+		log.Warn(ctx, "AuthType can be 0 or 1. Setting it to default value 0")
+		authType = 0
```

Comment:
don't hardcode; define a constant for defaultAuthType

Reply:
Used authTypeBasic instead of creating a new constant